### PR TITLE
[#19] ignore TS Modules

### LINKED ISSUE #19: TypeScript - import from declare module is moved to top
I have `global.d.ts` with contents:

```ts
declare module "*.scss" {
  const content: { [className: string]: string };
  export = content;
}

declare const env: {
  [key: string]: string;
};


declare module "path-browserify" {
  import * as path from "path";
  export = path;
}
```

and after running prettier on this file this happens:

```diff
+  import * as path from "path";
+
declare module "*.scss" {
  const content: { [className: string]: string };
  export = content;
}

declare const env: {
  [key: string]: string;
};


declare module "path-browserify" {
-  import * as path from "path";
  export = path;
}
```

### PR DESCRIPTION
**Pull request by byara**
Fixes #19

### CODE REVIEW

**Review by ayusharma — APPROVED** [review]
Very nice

File: src/preprocessor.ts [hunk 1]
```diff
@@ -21,7 +24,10 @@ export function preprocessor(code: string, options: PrettierParserOptions) {
 
     traverse(ast, {
         ImportDeclaration(path: NodePath<ImportDeclaration>) {
-            importNodes.push(path.node);
+            const tsModuleParent = path.findParent((p) => isTSModuleDeclaration(p));
```
Comment (ayusharma) [inline src/preprocessor.ts:27]: can we check this after traversing AST ?  May be we can short-circuit for the *.d.ts files

Comment (byara) [inline src/preprocessor.ts:27]: I'm not really sure that we can do that.

Comment (byara) [reply src/preprocessor.ts:27]: I'm not really sure that we can do that 🤔

Comment (byara) [reply src/preprocessor.ts:27]: Also, do you see any other case .d.ts files that this change might break it?

**Pull request merged**